app/controllers/draws_controller.rb

```diff
@@ -52,6 +52,11 @@ def filter_intent_report
     render action: 'intent_report'
   end
 
+  def suite_summary
+    @all_sizes = SuiteSizesQuery.call
+    @suites_by_size = SuitesBySizeQuery.new(@draw.suites).call
```

Comment:
also, I know we talked about this yesterday, but I still don't see how this excludes suites with groups